packages/swc-configs/src/jest.ts

```diff
@@ -2,20 +2,26 @@ import type { Config as SwcConfig } from "@swc/core";
 import { applyTransformers, type SwcConfigTransformer } from "./applyTransformers.ts";
 
 export interface DefineJestConfigOptions {
-    react?: boolean;
+    baseUrl?: NonNullable<SwcConfig["jsc"]>["baseUrl"];
     parser?: "ecmascript" | "typescript";
+    paths?: NonNullable<SwcConfig["jsc"]>["paths"];
+    react?: boolean;
     transformers?: SwcConfigTransformer[];
 }
 
 export function defineJestConfig(options: DefineJestConfigOptions = {}) {
     const {
-        react = false,
+        baseUrl,
         parser = "typescript",
+        paths,
+        react = false,
```

Comment:
react is not used here?

Reply:
Not by default as you might want to test a library of code that isn't developped in React.

Reply:
i mean the variable is deconstructed, but never used anywhere